dbx/commands/deploy.py

```diff
@@ -405,10 +405,13 @@ def _walk_content(func, content, parent=None, index=None):
 def _adjust_path(candidate, adjustment, file_uploader: FileUploader):
     if isinstance(candidate, str):
         # path already adjusted or points to another dbfs object - pass it
-        if candidate.startswith("dbfs"):
+        if candidate.startswith("dbfs") or candidate.startswith("/dbfs"):
```

Comment:
Seems like this would be backwards incompatible.

ex: If I had a folder locally named "dbfs" and I am referencing paths like `/dbfs/path/to/file.py`, then those files would currently be adjusted, uploaded, and referenced.

Where as, with your change, any paths starting with "/dbfs" will be skipped.